dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/pom.xml

```diff
@@ -125,26 +125,23 @@
                     </execution>
                 </executions>
             </plugin>
+        </plugins>
+    </build>
+
+    <reporting>
+        <plugins>
             <plugin>
-                <artifactId>maven-site-plugin</artifactId>
-                <version>3.4</version>
+                <artifactId>maven-project-info-reports-plugin</artifactId>
+                <version>2.4</version>
```

Comment:
I used the same plugin version (just moved the plugin from one section to another). 
Do you think we should upgrade it?